### DISCUSSION TITLE
How to run the CheckStyle?

### DISCUSSION THREAD

**Discussion by vic816** [CastleLab/COMP3021-2021Fall-PA1-Student-Version#26] [Q&A] [answered]
After reading the README, I am still unsure how to edit the run configurations to run the CheckStyle. 

Can you please provide me with the way of doing the configuration?

**Comment by Derppening** [comment, marked as answer]
You will see a bar like this at the top-right corner.

![image](https://user-images.githubusercontent.com/4293438/135297453-ecb11455-0446-473b-ac71-cf8e0630d064.png)

Click on the dropdown, and select "CheckStyle".

![image](https://user-images.githubusercontent.com/4293438/135297573-280318f6-f75f-4d29-8652-342b0ba7cea9.png)

Click on the "Run" button beside the dropdown.

![image](https://user-images.githubusercontent.com/4293438/135297749-f01f1fcf-3d57-44d0-90ea-c1903f7c5bfa.png)

A report will be generated in `app/build/reports/checkstyle`.

If you don't see the "CheckStyle" configuration, download the skeleton code again, and copy everything in `.idea/runConfigurations` into your project in the same path.
